### ISSUE TITLE
Inconsistency in writing file names to rda-file

### ISSUE THREAD

**Issue by ppdewolf** [sdcTools/UserSupport#47]
**SDC tool used:** tau-argus
**Version used:**  4.1.4
**Operating system used:** Windows

---
Saving a table as "intermediate", the name of the codelist-file does not contain directory name, hierarchy-file does have directory name in the .rda-file. E.g., (excerpt):

```
Region "99"
    <RECODEABLE>
    <TOTCODE> "Total"
    <DISTANCE> 2 4 4 4 4
    <CODELIST> "REGION.CDL"
    <HIERARCHICAL>
    <HIERCODELIST> "E:\Users\Peter-Paul\MyDocuments\data\region2.hrc"
    <HIERLEADSTRING> "@"
```

**Comment by AncoHundepool** [comment]
I think that the file name is exactly the same as in the RDA file when you did read the microdata?
However we can always add the full path for a file name  in a RDA that we generate..

**Comment by ppdewolf** [comment]
Solved in 4.1.6 build 2
